codemon/CodemonHelp.py

```diff
@@ -14,12 +14,12 @@ def showHelp():
   
   print("A CLI tool to ace competitive programming contests")
   print(colored.cyan("\nCOMMANDS: \n"))
-  print("codemon - - - - - - - - - - - - - - -  displays this message")
-  print("codemon --help - - - - - - - - - - - - shows help")
-  print("codemon init <contestName> - - - - - - initialises a contest")
-  print("codemon fetch <contestName> - - - - fetches all the sample test cases for the contest name provided")
-  print("codemon fetch  - - - - - - - - - - - extracts contest name from contest directory name and fetches all sample test cases ")
-  print("codemon init -n <file> - - - - - - - - creates file with given name")
-  print("codemon listen - - - - - - - - - - - - compiles and gives output")
-  print("codemon reg - - - - - - - - - - - - -  Register user details")
-  print("codemon practice <dirName> - - - - - - Starts a practice session")
+  print("codemon - - - - - - - - - - - - - - - -  Shows help")
+  print("codemon --help - - - - - - - - - - - - - Shows help")
+  print("codemon init <contestName> - - - - - - - Initialises a contest")
+  print("codemon fetch <contestName> - - - - - -  Fetches sample TC from CF for <contestName> round")
```

Comment:
TC from CF is not proper. It can be difficult for people to know what TC is and what CF is.